review.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+// Conduct a brief Code Review of the given code snippets in the context of the previous Task
+
+/*
+ * The function storyStats returns four things:
+ * the shortest word
+ * the longest word
+ * the average word length
+ * the list of words that are of length equal to the average word rounded up and down, or empty list if such words do not exist.
+ */
+func storyStats1(str string) (map[string]string, error) {
+	var err error
+	m := make(map[string]string)
+	m["shortest"] = ""
+	m["longest"] = ""
+	m["average"] = ""
+
+	str = strings.TrimSpace(str)
+	if str == "" {
+		err = errors.New("can't work empty string")
+		return nil, err
+	}
+	var shortest int
+	var longest int
+	words := strings.Fields(str)
+
+	for index, element := range words {
+		if index == 0 {
+			// set intail values
+			shortest = len(element)
+			longest = len(element)
+			m["shortest"] = element
+			m["longest"] = element
+			continue
+		}
```

Comment:
Simplify it, setup default values for shortest(math.MaxInt32) and for longest(0)